### ISSUE TITLE
finding vulnerable subdomains out of found subdomains

### ISSUE THREAD

**Issue by 0xPrateek** [0xPrateek/Stake#1]

**Comment by gr33nm0nk2802** [comment]
Hey, I would like to work on this issue

**Comment by 0xPrateek** [comment]
Yes you can start to work on this issue. I am assigning you this issue. 
Stake does two things : - Subdomain enumeration and then - Subdomain takeover detection 
Make sure you are aware of the subdomain takeover bug and know how to reproduce that bug then only you can automate this process.

**Comment by 0xPrateek** [comment]
Before I assign you this, can you describe how you will do it ?

**Comment by gr33nm0nk2802** [comment]
I would like to take 2 days time before being assigned to this issue. Let me get familiar with this bug and try to implement a bit before contributing into this. Thanks I have also gone through the blog post 
https://0xpatrik.com/subdomain-takeover-basics/
still I need to see a working example of the process before getting into it. Thanks

**Comment by gr33nm0nk2802** [comment]
All i understood about Subdomain Takeover is , when the browser asks for the a subdomain (say target.com) address from DNS server, A recursive DNS resolver sends a request to the DNS server of target.com. Here, target.com has a cannonical name to another subdomain (say vulnerable.com). The site target.com may be registered and has several subdomains in its CNAME record but, the subdomain, vulnerable.com is available for registration. So, the attacker may register to this subdomain and till the time the CNAME record is available a user trying to access  target.com maybe redirected to the new server IP rather than the actual IP address of the server.

**Comment by 0xPrateek** [comment]
Good to hear that you will learn about the issue first. I am dropping some links which will help you and if you want help you can ping me in the official communication channel and I'll help you with the bug.
Here are some great links to look for subdomain takeover bug;
- [Can I take over XYZ?](https://github.com/EdOverflow/can-i-take-over-xyz)
- [Subdomain takeover POC ](https://www.youtube.com/watch?v=GHy6bBe6dJE)
- [Complete Blog of Patrik](https://0xpatrik.com)
